docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -939,7 +937,7 @@ def is_reserved_suffix(suffix_map, suffix):
 #  Label processing helper functions
 ############################
 def remove_multiple_appearance_indicator(label):
-  return re.sub('_{2,}\d+', '', label)
+  return re.sub(r'_{2,}\d+', '', label)
```

Comment:
Also curious about this change. Functional, or aesthetic?

Reply:
Linter flagged this. \d is an escape code--should be a raw string